### ISSUE TITLE
Bug in Silver Table Code

### ISSUE THREAD

**Issue by nickaustinlee** [Labelbox/labelspark#22]
I tried to convert a bronze table to silver where the Label.classification.answer column (from flattened bronze table) looked like this: 

`
[[{"featureId": "redacted_for_confidentiality", "schemaId": "redacted_for_confidentiality", "title": "Response", "value": "response"}], [{"featureId": "redacted_for_confidentiality", "schemaId": "redacted_for_confidentiality", "title": "Response 2", "value": "response"}], [{"featureId": "redacted_for_confidentiality", "schemaId": "redacted_for_confidentiality", "title": "Response 3", "value": "response"}], [{"featureId": "redacted_for_confidentiality", "schemaId": "redacted_for_confidentiality", "title": "Response 4", "value": "response"}]]`

As you can see, it is an array of arrays.

It seemed that the add_json_answers_to_dictionary() method failed to process the flattened label column properly. The method iterated through the array of Label.classifications.answer but the ast literal parsing seemed to get stuck after that. Also I think that since the method was designed to parse JSON strings, it was failing to process the Row. 

This modification seemed to fix the method (simply tacked the code on at the end), but further testing is necessary. 

```
#this runs if the literal stuff didn't run and it's not json 
    if isinstance(answer, list) and len(answer) == 1: 
      answer = answer[0] #puzzled how we get a list of length 1 where the contents is the row 
      my_dictionary[title] = answer["title"]
    else: 
      my_dictionary[title] = answer
```

I also believe we need to revamp add_json_answers_to_dictionary() to make it easier to debug, as it consists of some relatively brittle code built around edge cases encountered during the many different JSON scenarios we get back from the Labelbox API due to nesting and such.

**Comment by nickaustinlee** [comment]
I have done further investigation and found an issue with the way the library currently handles the JSON export format for dropdown classifications. Radio classifications seem to work just fine. Further investigation is necessary to identify edge cases, and also while we are at it we can investigate support for multi-classification support (e.g. multiple items selected in a checklist or dropdown).

**Comment by nickaustinlee** [comment]
We are beta testing the fix for this right now --it should be pushed to master within a few business days if tests go well. 

Anybody who wants to try out the Release Candidate can build a wheel from the nl/classifications_hotfix branch

**Comment by nickaustinlee** [comment]
This has been fixed in 0.4.0 which is now merged.
